charts/flagger/crds/crd.yaml

```diff
@@ -1191,6 +1191,7 @@ spec:
                     - msteams
                     - discord
                     - rocket
+                    - gchat
```

Comment:
Please add this to `artifacts/flagger/crd.yaml` then run `make crd` and commit all changes.

Reply:
Done